pyanaconda/core/configuration/anaconda.py

```diff
@@ -105,6 +105,111 @@ def _is_unknown(self):
         """Are we running in the unknown OS?"""
         return self._type is SystemType.UNKNOWN
 
+    @property
+    def can_reboot(self):
+        """Can we reboot the system?"""
+        return self._is_boot_iso
+
+    @property
+    def can_switch_tty(self):
+        """Can we change the foreground virtual terminal?"""
+        return self._is_boot_iso
+
+    @property
+    def can_audit(self):
+        """Can we run the audit daemon?"""
+        return self._is_boot_iso
+
+    @property
+    def can_adjust_time(self):
+        """Can we change the time?"""
+        return self._is_boot_iso
+
+    @property
+    def can_adjust_time_live(self):
+        """Can we change the time?
+
+        FIXME: Conflict with can_adjust_time.
+        """
+        return self._is_boot_iso or self._is_live_os
+
+    @property
+    def can_synchronize_time(self):
+        """Can we run the NTP daemon?"""
+        return self._is_boot_iso
+
+    @property
+    def can_localize(self):
+        """Can we change the localization?"""
+        return self._is_boot_iso
+
+    @property
+    def can_localize_live(self):
```

Comment:
I'm thinking if these shouldn't be just two checks? You can test if this can be localized and if this installation is on Live installation. I would just remove this completely.

Reply:
Changed to `can_configure_keyboard`.